typescript/sdk/src/ism/HyperlaneIsmFactory.ts

```diff
@@ -596,6 +664,63 @@ export async function moduleMatchesConfig(
   return matches;
 }
 
+export async function routingModuleDelta(
+  destination: ChainName,
+  moduleAddress: Address,
+  config: RoutingIsmConfig,
+  multiProvider: MultiProvider,
+  contracts: HyperlaneContracts<ProxyFactoryFactories>,
+  mailbox?: Address,
+): Promise<RoutingIsmDelta> {
+  const signer = multiProvider.getSigner(destination);
```

Comment:
I would remove signer and `isOwner` from this

Reply:
especially if this is "delta"

Reply:
what did you decide here? this function is useful without having a signer available

Reply:
removed isOwner